contributors.py

```diff
@@ -0,0 +1,87 @@
+import urllib.request

```

Comment:
Although this file has immense utility, it should not be a part of the repository